### PR TITLE
Add option to manually broadcast raid layout message

### LINKED ISSUE #9541: Change party raid scout broadcast to be manual
Instead of spamming every single raid you scout, there should be a button on the screen when you enable the feature, so that once you've actually found a solid scout, you can choose to broadcast *that* raid to the discord party.

Right now, the feature is basically un-usable.

### PR DESCRIPTION
**Pull request by deathbeam**
Closes #9541

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

**Comment by Adam-** [comment]
I don't see how this addresses the concern in #9541 of it just spamming uninteresting raids, since this still does that. I'd be okay with it just never broadcasting unless you click broadcast as suggested.

File: runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java [hunk 1]
```diff
@@ -252,6 +254,16 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 	}
+	@Subscribe
+	public void onOverlayMenuClicked(final OverlayMenuClicked event)
+	{
+		if (event.getEntry().getMenuAction() == MenuAction.RUNELITE_OVERLAY &&
+			event.getEntry().getOption().equals(RaidsOverlay.BROADCAST_ACTION) &&
+			event.getOverlay() instanceof RaidsOverlay)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java]: this is also just `event.getOverlay() == overlay`

**Comment by deathbeam** [comment]
Because you can simply disable that, it's config option @Adam- . And this manual broadcast works always.

**Comment by deathbeam** [comment]
Sure why not

**Pull request merged**